### PR TITLE
Copter: Notify GCS that Flight mode change failed

### PR DESCRIPTION
**Pull request by lvale**
Copter: Notify GCS that Flight mode change has failed

### CODE REVIEW

File: ArduCopter/flight_mode.cpp [hunk 1]
```diff
@@ -129,6 +129,7 @@ bool Copter::set_mode(control_mode_t mode, mode_reason_t reason)
     }else{
         // Log error that we failed to enter desired flight mode
         Log_Write_Error(ERROR_SUBSYSTEM_FLIGHT_MODE,mode);
+        gcs_send_text(MAV_SEVERITY_WARNING,”Flight mode change failed");
```
Comment (OXINARF) [inline ArduCopter/flight_mode.cpp]: The first quote mark is wrong, can you fix that? Otherwise I think whoever merges this can fix that.

Comment (lvale) [inline ArduCopter/flight_mode.cpp]: Damn web interface that has a mind of itself...Damn autocorrect...

**Comment by lvale** [comment]
@rmackay9 @OXINARF  something against pushing this, or do I close it ?

**Comment by OXINARF** [comment]
Needs to be squashed, but other than that I have nothing against it - I'm completely in favor actually.

@rmackay9 Are you OK with this?

**Comment by rmackay9** [comment]
Yes, seems like a good change to me.

**Comment by OXINARF** [comment]
Squashed and merged, thanks!

**Pull request closed without merging**